src/commands/utility/info/category.js

```diff
@@ -0,0 +1,39 @@
+const { Command, SwitchbladeEmbed, CommandError } = require('../../../')
+const moment = require('moment')
+
+module.exports = class InfoCategory extends Command {
+  constructor (client) {
+    super({
+      name: 'category',
+      parent: 'info',
+      requirements: { guildOnly: true },
+      parameters: [{
+        type: 'string',
+        required: true,
+        full: true,
+        missingError: 'commands:info.subcommands.voice.missingChannelName'
+      }]
+    }, client)
+  }
+
+  async run ({ message, t, author, channel, language }, givenChannel) {
+    moment.locale(language)
+    givenChannel = message.guild.channels.cache
```

Comment:
Use `find` and don't reassign a param
```suggestion
    const foundChannel = message.guild.channels.cache.find(channel => channel.type === 'category' && channel.name.toLowerCase() === givenChannel.toLowerCase());
```